src/main/scripts/org/reaktivity/specification/nukleus/control/resolve/with.roles/nukleus.rpt

```diff
@@ -0,0 +1,44 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+property nukleiR05N ${nuklei:directory("target/nukleus-itests").controlCapacity(1024, 1024)}
+
+property controlR05N ${nukleiR05N.controlNew("security")}
+
+property securityNukleusId [0x01 0x00]
+property roleBits [0x0c 0x00 0x00 0x00 0x00 0x00]
+
+connect "agrona://stream/bidirectional"
+        option agrona:reader ${agrona:manyToOneReader(controlR05N.nukleus)}
+        option agrona:writer ${agrona:broadcastTransmitter(controlR05N.controller)}
+
+connected
+
+read [0x00 0x00 0x00 0x03]
+read ([0..8]:correlationIdR05N)
+read [0x05] "role1"
+read [0x05] "role2"
+
+write [0x40 0x00 0x00 0x03]
+write ${correlationIdR05N}
+write ${securityNukleusId}
+write ${roleBits}
```

Comment:
Local variables should use same suffix as `correlationId`, ie. `R05N` in this case.

Rename {securityNukleusId, roleBits} to {authorizationScope, authorization}

Reply:
Ok. I enter for a single variable called authorizationR05N to avoid confusion given that the whole 8 bytes I referred to as authorization in the IDL